codes/data/multi_country/quality-check-processed-datasets.py

```diff
@@ -0,0 +1,130 @@
+import pandas as pd
+import numpy as np
+from codes import utils as cutil
+
+#### Settings
+
+path_cutoff_dates = cutil.HOME / 'codes' / 'data' / 'cutoff_dates.csv'
+cutoff_dates = pd.read_csv(path_cutoff_dates)
+cutoff = pd.to_datetime(
+    str(cutoff_dates.set_index('tag').loc['default', 'end_date'])
+)
+use_cutoff = cutil.PROCESSED_DATA_DATE_CUTOFF
+
+path_template = cutil.DATA_PROCESSED / '[country]_processed.csv'
+
+template = pd.read_csv(path_template)
+
+country_codes = {
```

Comment:
Some countries produce multiple adm-levels. Might as well check them all. So you could loop through 0,1,2 and through each iso3 code, and just if the dataset exists, then check it. If not, then move onto the next.

Reply:
Good point! will do